src/session/session_api.c

```diff
@@ -2484,17 +2492,22 @@ __open_session(WT_CONNECTION_IMPL *conn, WT_EVENT_HANDLER *event_handler, const
     if (config != NULL)
         WT_ERR(__session_reconfigure((WT_SESSION *)session_ret, config));
 
+    /* Initialize the default error info, including a buffer for the error message. */
+    WT_ERR(__wt_calloc_one(session, &err_msg_buf));
+    WT_ERR(__wt_buf_init(session, err_msg_buf, err_msg_buf_size));
+    F_SET(err_msg_buf, WT_ITEM_INUSE);
+
+    session_ret->err_info.err_msg = NULL;
+    session_ret->err_info.err_msg_buf = err_msg_buf;
+    F_SET(session_ret, WT_SESSION_SAVE_ERRORS);
+    WT_ERR(__wt_session_set_last_error(session_ret, 0, WT_NONE, WT_ERROR_INFO_EMPTY));
```

Comment:
I wonder if should just explicitly just set the individual components here rather than calling the function, it would decouple the need to have WT_ERROR_INFO_EMPTY inside as well.

Reply:
Oh yes you are correct. In that case ignore this.